dyno-contrib/src/main/java/com/netflix/dyno/contrib/EstimatedHistogramBasedCounter.java

```diff
@@ -83,13 +83,19 @@ public Number getValue() {
 			return estHistogram.mean();
 		}
 
+        @Override
+        public Number getValue(int pollerIndex) {
```

Comment:
Seems to be a new concept added in Servo to keep track of the poller interval when recording more advanced metrics. I believe @brharrington could give a better explanation and help with a better implementation if anyone thinks there is value in it.